### ISSUE TITLE
remote-tidy: don't block shutdown if a host cannot be contacted

### ISSUE THREAD

**Issue by oliver-sanders** [cylc/cylc-flow#5181]
> [Reported on Element](https://matrix.to/#/!hwZqSYihGPuhDdIzIP:matrix.org/$1665066472327imWSK:matrix.org?via=matrix.org)

Ensure that Cylc is able to shut down properly even if one or more platforms are not contactable for remote-tidy purposes.

Reported issue where a Cylc process remained running long after the `DONE` message, possibly related to remote-tidy? Needs verification:

```
2022-10-06T00:33:43Z ERROR - Error during shutdown
2022-10-06T00:33:43Z ERROR - NoHostsError: Unable to find valid host for mustang
2022-10-06T00:33:43Z INFO - DONE
```

If a platform goes down, Cylc should log the issue, but allow remote-tidy to be skipped for that platform.

We should be able to test this using a functional test and configuring `ssh command` to use a script which exits `255` when required for test purposes (e.g. by looking at an environment variable).

**Comment by oliver-sanders** [comment]
I think this may have been fixed by changes in 8.1.0, needs confirmation. To test, try getting Cylc to submit jobs to remote platforms, then make those platforms un-contactable (e.g. scramble SSH), then shut the workflow down.

**Comment by oliver-sanders** [comment]
@wxtim, assigning you as I think your PR will close this. This issue is specifically about the failure of remote-tidy during shutdown.

**Comment by oliver-sanders** [comment]
Closed by #5445
